R/pkg/R/SQLContext.R

```diff
@@ -343,7 +343,6 @@ setMethod("toDF", signature(x = "RDD"),
 #' path <- "path/to/file.json"
 #' df <- read.json(path)
 #' df <- read.json(path, multiLine = TRUE)
-#' df <- jsonFile(path)
```

Comment:
@felixcheung maybe you can check my work here. Is this the right amount of stuff to delete from SparkR when removing these deprecated SQLContext methods?

Reply:
yes, that's part of it, I want to remove other deprecated methods also